plugins/wasmembed/abi.hpp

```diff
@@ -1,4 +1,14 @@
+#ifdef __has_include
```

Comment:
I'm not sure this is needed, and may cause problems if it isn't implemented as a macro (I think it was historically, but now part of the standard.)  We compile with c++17 so it will be supported.

Reply:
I agree - it is new to me and my first draft had the `else` inside the inner `#if`.  

I will change and if the CentOS 7 passes we should be good.